inc/Config/QueryRunner/QueryRunner.php

```diff
@@ -272,17 +275,10 @@ public function execute_batch( HttpQueryInterface $query, array $array_of_input_
 
 		if ( 1 === count( $id_list_input ) ) {
 			$id_list_slug = array_key_first( $id_list_input );
-			$ids = array_reduce(
-				array_column( $array_of_input_variables, $id_list_slug ),
-				function ( array $carry, mixed $item ): array {
-					if ( is_array( $item ) ) {
-						return array_merge( $carry, $item );
-					}
-
-					return array_merge( $carry, [ $item ] );
-				},
-				[]
-			);
+			$ids = [];
```

Comment:
Replacing the `array_reduce` had a minor impact on the execution of the queries, hence why I kept this in.

Reply:
I collected the timings of the minor impact changes together, and used that as the basis for keeping em tbh. I dropped a lot of the other minor ones as they weren't worth doing. This one I kept, as you said, it makes it cleaner.

That said, I'm going to revert it. I'm just going over all the other performance improvements and dropping em if it's not really needed.